README.md

```diff
@@ -81,6 +81,7 @@ While designing distributed systems are hard enough, testing them is even harder
 - [ETH Zurich: Distributed Systems](http://dcg.ethz.ch/lectures/podc_allstars/)
 - [ETH Zurich: Distributed Systems Part 2](http://dcg.ethz.ch/lectures/distsys), covers  Distributed control algorithms, communication models, fault-tolerance among other things. In particular fault tolerance issues (models, consensus, agreement) and replication issues (2PC,3PC, Paxos), which are critical in understanding distributed systems are explained in great detail.
 - [Distributed Systems Course](http://www.distributedsystemscourse.com/), A beginner course on distributed system by Chris Colohan, A google employee who contributed to  SUIF, MapReduce, TCMalloc, Percolator, Caffeine, Borg, Omega, and Piper.
+- [MIT 6.824](https://www.youtube.com/playlist?list=PLrw6a1wE39_tb2fErI4-WkMbsvGQk9_UB), MIT distributed system lectures, in each video they discuss papers like GFS, Zookeeper, RAFT, Spanner...
```

Comment:
can you link to OCW first, so something like

[MIT 6.824](https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-824-distributed-computer-systems-engineering-spring-2006/) ..., and video lectures (yt link)...

Reply:
yess will do it

Reply:
added OCW link and youtube playlist https://github.com/theanalyst/awesome-distributed-systems/pull/38/commits/c8209deae306bb7fa025810647453e626b051eba